front/src/common/InfraSelector/InfraSelectorEditionActionsBar.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react';
-import { FaCopy, FaDownload, FaLock, FaLockOpen, FaPencilAlt } from 'react-icons/fa';
+import { GoDuplicate, GoPencil, GoDownload, GoLock, GoUnlock } from 'react-icons/go';
```

Comment:
```suggestion
import { GoDuplicate, GoPencil, GoDownload, GoLock, GoUnlock, GoXCircleFill,
  GoCheck, } from 'react-icons/go';
```
GoXCircleFill can be used instead of `MdCancel` at **line 125**
GoCheck can be used instead of `MdCheckl` at **line 133**